src/components/organisms/TodoList/TodoList.tsx

```diff
@@ -46,7 +46,7 @@ const TodoList = ({
                         onDeleteTodo={()=> onToBeDeleteTodo(todo)}
                         onUpdateTodo={() => onUpdateTodo(todo)}
                         isActiveKebab={todo.id === activeId}
-                        showKebab={!todo.isDone}
+                        showKebab={true}
```

Comment:
I think you can just do it like this
```suggestion
                        showKebab
```

Reply:
Noted on this @itsmitchyyy , I'll change this one.

Reply:
@itsmitchyyy 

Done addressing requested changes.